data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+	on complete
+		conversation
+			`	Anna is still in awe as you touch down on the ringworld. As the two of you open the hatch, you see yet another Heliarch agent in front of you.`
+			`	"Greetings, humans. For you, what can I do today?", asks the Heliarch.`
+			`	"This is my friend Anna. I'm getting her registered with the Heliarchs today.", you answer.`
+			`	"To reach the registration office, this way you must go.", replies the Heliarch, pointing to a hallway.`
+			`	You and Anna walk down the hallway as directed, until you spot a registration office. Anna walks into the registration office while you sit down on a nearby bench. After 15 minutes, Anna walks out and approaches you. "They finally registered me. I'm off to the spaceport now to eat something. Meet me there so we can discuss our next move."`
+
+mission "Fangvin: Struggling Author 3"
+	name "Back to <planet>"
+	description "Now that Anna Fangvin has completed the registration process, help her get a translation device on <destination>."
+	source "Ring of Friendship"
+	destination "Ki Patek Ka"
+	to offer
+		has "Fangvin: Struggling Author 2: done"
+	on offer
+		conversation
+			`Going through the spaceport looking for Anna, you finally find her at a restaurant serving food made for Saryds, eating from a plate of vegetables, her laptop next to the plate of food. She sees you and motions for you to take a seat.`
+			`	You sit down next to Anna. "Finally, something I can eat that isn't a mix of vomit and soup!"`
+			choice
+				`	"How good is it?"`
+					goto "it's very good"
+				`	"What did you usually eat back in human space?"`
+			`	"A lot. As an author, all I had to eat were instant noodles and takeout food as I never could find time in my schedule to cook at home. I was working hard on Beyond the Hyperdrive at the time. This food is better than any instant noodle brand I could ever find on Zug."`
+			choice
+				`	"What's our plan now?"`
+				`	"What are we going to do next?"`
+					goto "i want to get my book published here"
+			label "it's very good"
+			`	"It's very good. Probably better than anything I've had back on Zug."`
+			choice
+				`	"So, what do you want to do now?"`
+			label "i want to get my book published here"
+			`	"I want to get my book published here."`
+			choice
+				`	"Have you found a publishing company yet?"`
+			`	"No." Anna then begins complaining about how it was extremely difficult for her to communicate with both the restaurant staff and the publishing companies, who did not have a translation box.`
+			choice
+				`	"I can get you a translator."`
+			`	"But will I have to pay for it?"`
+			choice
+				`	"I'll help you get one. You probably won't need to pay for one."`
+			`	"Great! So, where will we get the translator?"`
+			`	"We'll go to <destination> to ask the Heliarchs for a translation box", you tell Anna. "Finish your meal, then go back to my ship. I'll be ready." You walk off to your ship.`
```

Comment:
Considering you're on the Ring of Friendship, the capital, it wouldn't be the case that it's difficult to communicate. It's true that not everyone would have a translation box, but many would have them, and even without them there are legions of Heliarchs around that'd serve as translators if needed. You could instead have it that the reason she couldn't find a publishing company around is perhaps the Ring of Friendship doesn't really have that kind of service readily available, as I imagine these types of entertainment purposes there would be restricted to the actual shops, as there's not much of a benefit to having the actual publishing office there, as it'd not be visited as much, and a proper shop would be worth more to capitalize on the very high traffic in the system and the ring. Perhaps have it that she browsed what book shops she could find, but was informed that for getting something published she'd need to take it to the publishing office, which isn't in the Ring of Friendship. With that, *then* we could bring the lack of a translator into play in the sense that they'd likely give the world and contact means (ie coalition email), but as that'd require a translator over a longer time for getting the text figured out, it'd be very unlikely someone'd be willing to just lend their translator while Anna sorts it out with the publisher. This isn't to say we'd go after a translator box, I don't think that's worthwhile including in this mission chain at all, it'd simply be a place where you could mention it more fittingly, if you want.
Given the on complete for this has you two looking for a publishing company, you'd need to change that. My suggestion is to cut the middleman and have this mission, Fangvin: Struggling Author 3, already direct you to Inmost Blue. You could try to re-incorporate some of the conversation from this mission's (mission 3) on complete, and the next mission's (mission 4) on offer, in this mission's (mission 3) on offer, if you'd like.